mod-spring-petstore/src/main/java/org/folio/petstore/controller/PetsController.java

```diff
@@ -1,59 +1,45 @@
 package org.folio.petstore.controller;
 
 import lombok.RequiredArgsConstructor;
-import org.folio.petstore.domain.dto.Pet;
-import org.folio.petstore.rest.resource.PetsApi;
+import org.folio.petstore.domain.dto.PetDTO;
+import org.folio.petstore.domain.service.impl.PetServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please do not use wildcards

Reply:
Changes added for OpenAPI specification. Requesting your review.